pkg/aggregator/demultiplexer.go

```diff
@@ -18,6 +18,7 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/epforwarder"
 	"github.com/DataDog/datadog-agent/pkg/forwarder"
 	"github.com/DataDog/datadog-agent/pkg/metrics"
+	ddruntime "github.com/DataDog/datadog-agent/pkg/runtime"
```

Comment:
What do you think about `agentruntime` instead ?

Reply:
Done